controllers/users/updateUserAvatar.js

```diff
@@ -0,0 +1,43 @@
+import path from "path";
+import fs from "fs/promises";
+import { v4 as uuidv4 } from "uuid";
+import { storeImage } from "../../config/multerConfing.js";
+
+import { isImageAndTransform } from "../../helpers/isImageAndTransform.js";
+import { updateUserAvatarURL } from "../../service/index.js";
+
+export async function updateUserAvatar(req, res, next) {
+  if (!req.file) {
+    return res.status(400).json({ message: "File isn't a photo" });
```

Comment:
W przyszłości będzie można jakoś bardziej sensownie obsłużyć tę sytuację i napisać użytkownikowi że nie zgadza się typ pliku i jakie są dozwolone. Wtedy trzeba byłoby to zrobić wewnątrz uploadMiddleware.